namedtensor/torch_helpers.py

```diff
@@ -342,7 +342,6 @@ def __dir__(self):
         "expm1",
         "float",
         "floor",
-        "fmod",
```

Comment:
Be sure to rebase from master so you get this line back.

Reply:
Oops. sorry this is fine.

Reply:
Well, I deleted this line because I think 'fmod' is already in the self._noshift_args list, and putting it here caused the test_noshift fail.